src/components/Button/Button.jsx

```diff
@@ -1,8 +1,8 @@
 import "./Button.css";
 
-export const Button = ({buttonType, isDisabled, onClickFunction, children}) => {
+export const Button = ({buttonType, isDisabled=false, onClickFunction, children}) => {
```

Comment:
When we do `isDisabled=false`, if `isDisabled` is not passed, it will be `false` by default